Tribler/community/gigachannel/community.py

```diff
@@ -114,6 +118,19 @@ def respond_with_updated_metadata(self, peer, md_list):
         if reply_blob:
             self.endpoint.send(peer.address, self.ezr_pack(self.NEWS_PUSH_MESSAGE, RawBlobPayload(reply_blob)))
 
+    def notify_discovered_metadata(self, md_list):
+        """
+        Notify the GUI through event notifier that new torrents and channels are discovered.
```

Comment:
Minor comment: since this is code in the core which is not necessarily dependent on a GUI, you don't have to mention the GUI in the comments here I think.